packages/overlay/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "@swc-uxp-wrappers/overlay",
+    "version": "1.0.0",
+    "publishConfig": {
+        "access": "public"
+    },
+    "description": "UXP wrapper of the @spectrum-web-components/overlay component",
+    "license": "Apache-2.0",
+    "repository": {
+        "type": "git",
+        "url": "https://git.corp.adobe.com/torq/swc-uxp-wrappers.git",
+        "directory": "packages/overlay"
+    },
+    "author": "Adobe Inc",
+    "main": "src/index.js",
+    "module": "src/index.js",
+    "type": "module",
+    "exports": {
+        ".": "./src/index.js",
+        "./package.json": "./package.json",
+        "./src/Overlay.js": "./src/Overlay.js",
+        "./src/overlay.css.js": "./src/overlay.css.js",
+        "./sp-overlay.js": "./sp-overlay.js"
+    },
+    "dependencies": {
```

Comment:
We should specify transitive dependency as well (those which we have written wrappers for)- https://github.com/adobe/spectrum-web-components/blob/v0.37.0/packages/overlay/package.json#L131

Reply:
added it.